sympy/solvers/diophantine/diophantine.py

```diff
@@ -436,6 +435,9 @@ def diophantine(eq, param=symbols("t", integer=True), syms=None,
     True then permutations of the base solution and/or permutations of the
     signs of the values will be returned when applicable.
```

Comment:
These parameters section should be reformatted since I see
![_C__Users_sylee_Documents_Github_sympy_doc__build_html_modules_solvers_diophantine html](https://user-images.githubusercontent.com/34944973/98187891-4aa5e780-1f55-11eb-9114-b21a4277b6c1.png)

Reply:
I guess this was fixed? I built the docs in this branch and it seems to be formatted fine.